### PR TITLE
Docs: add database queries log documentation

### PR DESCRIPTION
**Pull request by dilipbheda**
Fixes https://github.com/WordPress/openverse/issues/5299

## Description
I've updated the `debugging.md` file to include the `env` variables for enabling database queries.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.
- [x] I ran the DAG documentation generator (`ov just catalog/generate-docs` for catalog
      PRs) or the media properties generator (`ov just catalog/generate-docs media-props`
      for the catalog or `ov just api/generate-docs` for the API) where applicable.

### CODE REVIEW

File: documentation/api/guides/debugging.md [hunk 1]
```diff
@@ -69,3 +69,12 @@ This is the guide to debugging the API using VS Code. This uses Microsoft's
    [Visual Code debugger's official instructions](https://code.visualstudio.com/docs/editor/debugging)
    to better understand how to use the debugger interface to accomplish tasks
    like setting breakpoints, watching variables etc.
+
+## Debugging Database Queries
+
+To log all database queries, add the following `env` variables to your configuration
```
Comment (AetherUnbound) [inline documentation/api/guides/debugging.md:75]: I'd suggest rewording this to be a bit more specific:

```suggestion
To log all database queries, add the following variables to your `api/.env` file:
```

**Review by AetherUnbound — DISMISSED** [review]
Thanks for your contribution @dilipbheda! I have a suggestion that will hopefully make the documentation clearer 😄

File: documentation/api/guides/debugging.md (hunk 1 above)
Comment (dilipbheda) [reply documentation/api/guides/debugging.md]: @AetherUnbound Thank you for the suggestion, I've update the content and pushed the changes.

File: documentation/api/guides/debugging.md [hunk 2]
```diff
@@ -69,3 +69,13 @@ This is the guide to debugging the API using VS Code. This uses Microsoft's
    [Visual Code debugger's official instructions](https://code.visualstudio.com/docs/editor/debugging)
    to better understand how to use the debugger interface to accomplish tasks
    like setting breakpoints, watching variables etc.
+
+## Debugging Database Queries
+
+To log all database queries, add the following variables to your `api/.env`
+file:
+
+```
```
Comment (dhruvkb) [inline documentation/api/guides/debugging.md:78]: All `.env` files we use are strict subsets of the shell syntax. This is also [recommended by GitHub linguist](https://github.com/github-linguist/linguist/pull/4922).

```suggestion
```shell
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
This is good, thanks for the contribution @dilipbheda! The content is accurate as well.

Could you move the content to a separate file named `logging.md` and add it to the `index.md` file? That's because the content you have added is about logging in general, and not related to the debugger. This file is very specifically about debugging the API line-by-line using VS Code's built-in debugger (hence the note at the top).

File: documentation/api/guides/debugging.md (hunk 2 above)
Comment (dilipbheda) [reply documentation/api/guides/debugging.md]: @dhruvkb Thank you for the review, I've applied your suggestions and pushed the changes. Please review them when you can.

**Review by dhruvkb — APPROVED** [review]
